install/src/main/resources/aspects.bzl

```diff
@@ -591,15 +596,15 @@ def _get_cpp_target_info(target, ctx):
     if hasattr(ctx.rule.attr, "linkshared"):
         linkshared = ctx.rule.attr.linkshared
 
-    _print_fields(copts)
-    _print_fields(defines)
-    _print_fields(linkopts)
-    print(linkshared)
-
-    return []
+    return create_struct(
+        copts = copts,
```

Comment:
u can replace those `_get_target_info` calls with `getattr(ctx, "name", default_value)` the same for linkshared